contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -179,6 +180,9 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     if (fulfillmentFlatFeeNativePPM > 0 && fulfillmentFlatFeeLinkDiscountPPM >= fulfillmentFlatFeeNativePPM) {
       revert LinkDiscountTooHigh(fulfillmentFlatFeeLinkDiscountPPM, fulfillmentFlatFeeNativePPM);
     }
+    if (nativePremiumPercentage > 100 || linkPremiumPercentage > 100) {
```

Comment:
You could define a `MAX_PREMIUM_PERCENTAGE` constant similar to the `MAX_REQUEST_CONFIRMATIONS` above. Don't worry about gas cost as constants are replaced during compile time. Defining such a constant will make the boundaries for the config clearer as well.

Reply:
Makes sense!

Reply:
According to [this piece of code](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol#L525) I figured that 100 should be the upper bound limit. As for your question about the described situation and what should we do in this case, probably @jinhoonbang is a better person to answer that.

Reply:
Setting the premium percentage as 150% means we are charging 250% of the gas cost. I don't think the LINK / Native price matters here as we're still getting the same value just denominated in different tokens. I would think the ability to set the premium percentages differently for native or link acts another mechanism to incentive people to pay in one of them.

If we want to be very safe, we can set the limit to the max of `155`